hs_explore/migrations/0016_recommendedresource_rec_type.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
```

Comment:
Probably a good idea to collapse these. Load, delete all migrations except initial, makemigrations, migrate. Check in one migration.